scrapers/Only3xGirls.yml

```diff
@@ -0,0 +1,32 @@
+name: Only3xGirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - only3xgirls.com
+      - purebj.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="sv_update_title"]/h1
+      Date:
+        selector: //ul[@class="dotsbetween"]//span[contains(.,"since")]/following-sibling::text()
+        postProcess:
+          - parseDate: 01/02/2006
+      Performers:
+        Name: //ul[@class="dotsbetween"]//a
+      Details:
+        selector: //div[@class='set-desc']
+        concat: "\n\n"
+      Tags:
+        Name: //div[@class="content tag"]//a
+      Image: //img[@class="stdimage thumbs"]/@src0_2x | //img[@class="thumbs"]/@src
```

Comment:
this need to be more specific we get a lot of images
```yaml
Image: //span[@class="update_thumb"]/img[@class="stdimage thumbs"]/@src0_2x | //span[@class="update_thumb"]/img[@class="thumbs"]/@src
```
should work ok, needs more testing though